src/helix/client/client_ext.rs

```diff
@@ -105,30 +123,101 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
     /// # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
     /// # let token = twitch_oauth2::UserToken::from_existing(&client, token, None, None).await?;
     /// use twitch_api::{helix, types};
+    /// use futures::TryStreamExt;
     ///
-    /// let ids: &[&types::UserIdRef] = &["1234".into(), "4321".into()];
     /// let chatters: Vec<helix::channels::ChannelInformation> = client
-    ///    .get_channels_from_ids(ids, &token).await?;
+    ///    .get_channels_from_ids(&["1234", "4321"][..].into(), &token).try_collect().await?;
```

Comment:
```suggestion
    ///     .get_channels_from_ids(&["1234", "4321"][..].into(), &token).try_collect().await?;
```

The indentation was three spaces before, but should really be four.